### DISCUSSION TITLE
Dynamic workflow branch for the "issue_comment" trigger

### DISCUSSION THREAD

**Discussion by MiguelRipoll23** [community/community#9030] [Actions] [answered]
A drawback from the _issue_comment_ trigger is that you can't test a workflow that uses it unless you merge to your default branch, it would be great if it used the branch from the pull request for example and if no branch can be obtained from the context use the default branch as it does currently.

**Comment by ethomson** [comment, marked as answer]
Hi @MiguelRipoll23, thanks for the feedback.  I agree that this makes testing your workflows difficult - we'll give some thought to how to improve this in the new year.
